### PR TITLE
Merge "Consortium manager" Q progress

### PR DESCRIPTION
**Pull request by usavkov-epam**
## Purpose

https://issues.folio.org/browse/UICONSET-126

Merge implemented "Quesnelia" related functionality from the feature branch

---

Related PRs:
- https://github.com/folio-org/ui-consortia-settings/pull/25
- https://github.com/folio-org/ui-consortia-settings/pull/26
- https://github.com/folio-org/ui-consortia-settings/pull/28
- https://github.com/folio-org/ui-consortia-settings/pull/30
- https://github.com/folio-org/ui-consortia-settings/pull/31
- https://github.com/folio-org/ui-consortia-settings/pull/33

## Pre-Merge Checklist
Before merging this PR, please go through the following list and take appropriate actions.
- [x] I've added appropriate record to the CHANGELOG.md
- Does this PR meet or exceed the expected quality standards?
  - [x] Code coverage on new code is 80% or greater
  - [x] Duplications on new code is 3% or less
  - [x] There are no major code smells or security issues
- Does this introduce breaking changes?
  - [ ] If any API-related changes - okapi interfaces and permissions are reviewed/changed correspondingly
  - [x] There are no breaking changes in this PR.

If there are breaking changes, please **STOP** and consider the following:

- What other modules will these changes impact?
- Do JIRAs exist to update the impacted modules?
  - [ ] If not, please create them
  - [ ] Do they contain the appropriate level of detail?  Which endpoints/schemas changed, etc.
  - [ ] Do they have all they appropriate links to blocked/related issues?
- Are the JIRAs under active development?
  - [ ] If not, contact the project's PO and make sure they're aware of the urgency.
- Do PRs exist for these changes?
  - [ ] If so, have they been approved?

Ideally all of the PRs involved in breaking changes would be merged in the same day to avoid breaking the folio-testing environment.  Communication is paramount if that is to be achieved, especially as the number of intermodule and inter-team dependencies increase.

While it's helpful for reviewers to help identify potential problems, ensuring that it's safe to merge is ultimately the responsibility of the PR assignee.

### CODE REVIEW

**Review by NikitaSedyx — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -9,3 +9,9 @@
 * [UICONSET-6](https://issues.folio.org/browse/UICONSET-6) Add tenant code to tenant profile.
 * [UICONSET-18](https://issues.folio.org/browse/UICONSET-18) "Switch active affiliation" profile dropdown action.
 * [UICONSET-125](https://issues.folio.org/browse/UICONSET-123) Add `ui-consortia-settings` app icon.
+* [UICONSET-7](https://issues.folio.org/browse/UICONSET-7) Create "Consortium manager" interface.
+* [UICONSET-8](https://issues.folio.org/browse/UICONSET-8) Limit access to "Consortium manager" interface.
```
Comment (Dmitriy-Litvinenko) [inline CHANGELOG.md:13]: Could you please add UICONSET-8 to Related PRs?

**Review by Dmitriy-Litvinenko — COMMENTED** [review]
Hello @usavkov-epam , @NikitaSedyx Thank you

File: CHANGELOG.md (hunk 1 above)
Comment (NikitaSedyx) [reply CHANGELOG.md:13]: https://github.com/folio-org/ui-consortia-settings/pull/25
it's in description

Comment (NikitaSedyx) [reply CHANGELOG.md:13]: @Dmitriy-Litvinenko ^^ :)

**Review by Dmitriy-Litvinenko — APPROVED** [review]

**Comment by dltj** [comment]
https://github.com/folio-org/ui-consortia-settings/pull/35

**Pull request merged**